src/js/components/MultiSelect/MultiSelect.js

```diff
@@ -0,0 +1,680 @@
+              id={`selected-${iValue}`}
+            >
+              {children && child ? (
+                child
+              ) : (
+                <CheckBox
+                  disabled={isDisabled(indexOptions)}
+                  label={
+                    <Box alignSelf="center" width="100%" align="start">
+                      {iLabel}
+                    </Box>
+                  }
+                  key={iLabel}
+                  pad="xsmall"
+                  tabIndex="-1"
+                  checked={value.includes(iValue)}
+                />
+              )}
+            </SelectOption>
+          );
+        }
+        return undefined;
+      },
+      [
+        valueKey,
+        allOptions,
+        children,
+        dropButtonRef,
+        isDisabled,
+        labelKey,
+        onSelectChange,
+        optionValue,
+        value,
+      ],
+    );
+
+    // element to show, trumps inputValue
+    const selectValue = useMemo(() => {
+      if (valueLabel instanceof Function) {
+        if (value) return valueLabel(value);
+      } else if (valueLabel) return valueLabel;
+      else if (value.length > 0 && visibleSelection) {
+        return (
+          <>
+            <Box
+              width="100%"
+              role="listbox"
+              aria-multiselectable
+              a11yTitle="Selected Options"
+            >
+              {value &&
+                allOptions
+                  .filter(
+                    (i) =>
+                      value.indexOf(
+                        valueKey && valueKey.reduce ? applyKey(i, valueKey) : i,
+                      ) !== -1,
+                  )
+                  /* eslint-disable-next-line array-callback-return, 
+                  consistent-return */
+                  .map((i) => visibleValue(i))}
+              {showA11yDiv && (
+                <Box
+                  height="0px"
+                  width="0px"
+                  overflow="hidden"
+                  // announce when an item is removed from selected options
+                  aria-live="assertive"
+                >
+                  <Text>{showA11yDiv}</Text>
+                </Box>
+              )}
+            </Box>
+            {value && value.length > 5 && (
+              <Box alignSelf="start">
+                <Button
+                  onClick={onRequestOpen}
+                  size="small"
+                  label={`+ ${value.length - 5} more`}
+                />
+              </Box>
+            )}
+          </>
+        );
+      }
+      return undefined;
+    }, [
+      valueKey,
+      value,
+      valueLabel,
+      visibleSelection,
+      showA11yDiv,
+      allOptions,
+      onRequestOpen,
+      visibleValue,
+    ]);
+
+    // After announcing set showA11yDiv to undefined so it won't
+    // be read out again
+    useEffect(() => {
+      if (showA11yDiv !== undefined) {
+        setTimeout(() => {
+          setShowA11yDiv(undefined);
+        }, 1000);
```

Comment:
How did we choose 1000?

Reply:
I picked it as a starting point, but after testing more with the screenreader I realized I can reduce the amount of time here

Reply:
I realized for chrome we don't need a timeout, however safari requires a timeout for this so I left his at 1000. I can add a comment in the code if that would be helpful